modules/dnn/src/layers/lrn_layer.cpp

```diff
@@ -92,12 +92,10 @@ class LRNLayerImpl CV_FINAL : public LRNLayer
 
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
-        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019) {
+        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 ||
+            backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH) {
             return bias == (int)bias;
         }
-        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH) {
-            return type == CHANNEL_NRM && bias == (int)bias;
```

Comment:
Consider updating this condition instead of merging backends `if`s.
Because on next improvement you will need to split these `if`s again.

So lets keep one `if` per one `backend`.